process_request/tests.py

```diff
@@ -55,6 +67,39 @@ def test_machineuser(self):
         self.assertEqual(str(user), system)
 
 
+class TestHelpers(TestCase):
+
+    def test_get_container_indicators(self):
+        letters = ''.join(random.sample(string.ascii_lowercase, random.randint(2, 10)))
+        title = "Digital Object: " + letters
```

Comment:
I would call this `expected_title` or something more descriptive. And you should use format strings rather than concatenation like this.